UnityProject/Assets/Scripts/Health/Living/HumanHealth/PlayerHealth.cs

```diff
@@ -14,9 +15,31 @@ public class PlayerHealth : LivingHealthBehaviour
 	[SerializeField]
 	private MetabolismSystem metabolism;
 
+	/// <summary>
+	/// The percentage of players that start with common allergies.
+	/// </summary>
+	[SerializeField]
+	private int percentAllergies;
```

Comment:
Please assign values to `private` `[SerializedField]`s to shut the compiler up about the warnings. Assign a number or `default` here. I see there are quite a few instances of these through the files.

Reply:
On it.

Reply:
![image](https://user-images.githubusercontent.com/32009105/91659384-27664400-ea9d-11ea-843a-860f1427d95b.png)

That doesn't seem to reduce the number of warnings.  Why?  Will do it anyway.

Reply:
Done.  If you found more, please tell me.

Reply:
Creates warnings in unity

Reply:
Understood.  It's all done.  Resolve the convo if you're satistified.